x-pack/plugins/apm/server/lib/rum_client/get_page_view_trends.ts

```diff
@@ -51,6 +51,16 @@ export async function getPageViewTrends({
               }
             : undefined,
         },
+        ...(breakdownItem
+          ? {
+              topBreakdowns: {
+                terms: {
+                  field: breakdownItem.fieldName,
+                  size: 9,
```

Comment:
Did we choose `9` for UI limit purposes? I think it would be cleaner if we name this value in a local constant.

Reply:
++ will do that in refactor, it's used in many other places for legends, it would good to extract in constant